pkg/static/login.js

```diff
@@ -164,17 +164,42 @@
     }
 
     function requisites() {
-        function disableLogin(name) {
+        function showBypass(bypass) {
+            if (bypass) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17774-20221024-133408-3792979a-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)